assets/bingo/lang/pl_pl.json

```
@@ -341,16 +341,16 @@
 	"bingo.lobby.team_selection.sign.line3": "Drużynę",
 	"bingo.lobby.team_selection.sign.line4": "",
 
-	//"bingo.lobby.team_selection.join_red": "%s joined the red team.",
-	//"bingo.lobby.team_selection.join_orange": "%s joined the orange team.",
-	//"bingo.lobby.team_selection.join_yellow": "%s joined the yellow team.",
-	//"bingo.lobby.team_selection.join_lime_green": "%s joined the lime team.",
-	//"bingo.lobby.team_selection.join_dark_green": "%s joined the dark green team.",
-	//"bingo.lobby.team_selection.join_cyan": "%s joined the cyan team.",
-	//"bingo.lobby.team_selection.join_light_blue": "%s joined the light blue team.",
-	//"bingo.lobby.team_selection.join_dark_blue": "%s joined the blue team.",
-	//"bingo.lobby.team_selection.join_purple": "%s joined the purple team.",
-	//"bingo.lobby.team_selection.join_magenta": "%s joined the magenta team.",
+	"bingo.lobby.team_selection.join_red": "Gracz @s dołączył do czerwonej drużyny.",
+	"bingo.lobby.team_selection.join_orange": "Gracz @s dołączył do pomarańczowej drużyny.",
+	"bingo.lobby.team_selection.join_yellow": "Gracz @s dołączył do żółtej drużyny.",
+	"bingo.lobby.team_selection.join_lime_green": "Gracz @s dołączył do jasnozielonej drużyny.",
+	"bingo.lobby.team_selection.join_dark_green": "Gracz @s dołączył do ciemnozielonej drużyny.",
+	"bingo.lobby.team_selection.join_cyan": "Gracz @s dołączył do błękitnej drużyny.",
+	"bingo.lobby.team_selection.join_light_blue": "Gracz @s dołączył do jasnoniebieskiej drużyny.",
+	"bingo.lobby.team_selection.join_dark_blue": "Gracz @s dołączył do niebieskiej drużyny.",
+	"bingo.lobby.team_selection.join_purple": "Gracz @s dołączył do fioletowej drużyny.",
+	"bingo.lobby.team_selection.join_magenta": "Gracz @s dołączył do karmazynowej drużyny.",
```

```suggestion
	"bingo.lobby.team_selection.join_red": "Gracz %s dołączył do czerwonej drużyny.",
	"bingo.lobby.team_selection.join_orange": "Gracz %s dołączył do pomarańczowej drużyny.",
	"bingo.lobby.team_selection.join_yellow": "Gracz %s dołączył do żółtej drużyny.",
	"bingo.lobby.team_selection.join_lime_green": "Gracz %s dołączył do jasnozielonej drużyny.",
	"bingo.lobby.team_selection.join_dark_green": "Gracz %s dołączył do ciemnozielonej drużyny.",
	"bingo.lobby.team_selection.join_cyan": "Gracz %s dołączył do błękitnej drużyny.",
	"bingo.lobby.team_selection.join_light_blue": "Gracz %s dołączył do jasnoniebieskiej drużyny.",
	"bingo.lobby.team_selection.join_dark_blue": "Gracz %s dołączył do niebieskiej drużyny.",
	"bingo.lobby.team_selection.join_purple": "Gracz %s dołączył do fioletowej drużyny.",
	"bingo.lobby.team_selection.join_magenta": "Gracz %s dołączył do karmazynowej drużyny.",
```

Placeholder is messed up on these